cli/worker.rs

```diff
@@ -121,7 +121,6 @@ pub struct CliMainWorkerOptions {
   pub serve_port: Option<u16>,
   pub serve_host: Option<String>,
 }
-
```

Comment:
Can you please revert this line?

Reply:
Looks like this hasn't been reverted yet